solutions/cses-1736.mdx

```diff
@@ -0,0 +1,156 @@
+---
+id: cses-1736
+source: CSES
+title: Polynomial Queries
+author: Albert Ye
+---
+
+## Explanation
+**Time Complexity: $\mathcal{O}(n\log n)$**
```

Comment:
only `Time Complexity:` should be bolded, move under `## Implementation`

Reply:
bolded latex won't matter but yeah i'll move it